index.html

```diff
@@ -69,5 +69,130 @@ <h2 class="connect">LET'S CONNECT</h2>
+            </div>
+          </div>
+        </div>
+        <div class="cardWork br">
+          <picture id="snapshot_sec"> </picture>
+          <div class="left Block_sec">
+            <div class="primary text_sec">
+              <h1 class="project title_sec">Multi-Post <br>
+                Stories</h1>
+              <div >
+                <ul class="frame">
+                  <li class="client_tag_list"><h2 class="client_canopy">CANOPY</h2></li>
+                  <span class="counter"></span>
+                  <li class="client_tag_role"><h2 class="client_role_text">Back End Dev</h2></li>
+                  <span class="counter_tw"></span>
+                  <li class="client_tag_year"><h2 class="client_yr_text">2015</h2></li>
+                </ul>
+              </div>
+            </div>
+            <div class="client_txt_bd">
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <ul class="tag_list">
+              <li class="html_tag"><h2 class="html_txt">html</h2></li>
+              <li class="css_tag"><h2 class="css_txt">css</h2></li>
+              <li class="java_tag"><h2 class="java_txt">javaScript</h2></li>
+            </ul>
+            <div id="action">
+              <div class="butt">
+                <button type="button" class="btn_text">See Project</button>
+              </div>               
+            </div>
+          </div>
+        </div>
+        <div class="cardWork th">
+          <picture id="snapshot_th" ></picture>
+          <div class="left block_th">
+            <div class="primary text_th">
+              <h1 class=" project title_th">Tonic</h1>
+              <div >
+                <ul class="frame">
+                  <li class="client_tag_list"><h3 class="client_canopy">CANOPY</h3></li>
+                  <span class="counter"></span>
+                  <li class="client_tag_role"><h3 class="client_role_text">Back End Dev</h3></li>
+                  <span class="counter_tw"></span>
+                  <li class="client_tag_year"><h3 class="client_yr_text">2015</h3></li>
+                </ul>
+              </div>
+            </div>
+            <div class="client_txt_bd">
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <ul class="tag_list">
+              <li class="html_tag"><h4 class="html_txt">html</h4></li>
+              <li class="css_tag"><h4 class="css_txt">css</h4></li>
+              <li class="java_tag"><h4 class="java_txt">javaScript</h4></li>
+            </ul>
+            <div id="action">
+              <div class="btnn">
+                <button type="button" class="btn_text" >See Project</button>
+              </div>               
+            </div>
+            </div>
+          </div>
+        </div>
+        <div class="cardWork fr">
+          <picture id="snapshot_fr" ></picture>
```

Comment:
Kindly, the PICTURE tag is used to wrap various versions of images, low quality, other formats compatible with other browsers, etc. In this case, you must use the "IMG" tag and indicate the attribute (example) alt="main logo"